content/questions/actions/question-116.md

```diff
@@ -0,0 +1,12 @@
+---
+question: "Select correct statements about approving workflow runs from public forks (Select two.)"
+archetype: "questions"
+title: "Question 116"
+draft: false
+---
+
+> https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks
+- [x] By default, all first-time contributors require approval to run workflows. 
+- [x] You can configure workflow approval requirements for a repository, organization, or enterprise.
+- [ ] When a pull-request is created from public forks, workflows can be run without approvals 
```

Comment:
This answer is tricky - could be marked as true as well in my eyes. 

Can you modify it to be crystal clear correct or wrong please?
![image](https://github.com/FidelusAleksander/ghcertified/assets/63016446/cf9133a6-98da-46c1-9b70-f826b50d5372)

Reply:
updated

Reply:
You updated the wrong answer - I always put comments on the exact line(s) that I'm commenting on :) 

I meant  this answer `When a pull-request is created from public forks, workflows can be run without approvals ` is tricky - because if the first checkbox is selected (from screenshot) then in theory workflows can be run without approvals - with the exception of newly created GitHub accounts

The screenshot i pasted is not the default setup. The second check is default If i remember correctly - as you previously stated in the correct answer.

Reply:
tl;dr modify `When a pull-request is created from public forks, workflows can be run without approvals` so it's crystal clear

Reply:
sorry, I did not get what you wanted. I have removed this question